File: C#/bubblesort.cs
using System;
class Sort{

    public static void Main(){
        int[] numbers = new int[5];
        int input;

        for (int i = 0; i<numbers.Length; i++){
            input=Convert.ToInt32(Console.ReadLine());
            numbers[i]=input;
        }
        int temp;
        for (int x=0;x<numbers.Length-1; x++){
            for (int y=x+1;y<numbers.Length;y++){
                if(numbers[x]>numbers[y]){
                    temp=numbers[x];
                    numbers[x]=numbers[y];
                    numbers[y]=temp;
                }
            }
        }

        Console.WriteLine("Your Array Sorted:");
        foreach (int num in numbers)
        {
            Console.WriteLine(num);
        }
    }
}

File: C/bubblesort.c
/*
*I'm a Programm3r, I have no lif3! -Stan (o_X)
*/

#include <stdio.h>
#define MAX 5

int myArray[MAX];

void Sort(){
	int i, j;
	for (i = 0; i < MAX - 1; i++){
		for (j =i+1; j < MAX; j++){
			if (myArray[i] > myArray[j]){
				int temp = myArray[i];
				myArray[i] = myArray[j];
				myArray[j] = temp;
			}
		}
	}
	
	for (i = 0; i < MAX; i++){
		printf("%d ", myArray[i]);
	}
}

int main(){
	
	printf("Enter any 5 numbers:\n");
	int i, num;
	for (i = 0; i < MAX; i++){
		scanf("%d", &num);
		myArray[i] = num;
	}
	Sort();
  return 0;
}

